### DISCUSSION TITLE
服务器转发模式运行，有时候会在日志里输出这堆东西

### DISCUSSION THREAD

**Discussion by SSSSSSSan** [xfgryujk/blivechat#194] [提问] [answered]
![image](https://github.com/xfgryujk/blivechat/assets/40064285/fdce95ef-8d8a-4c10-b51c-c35477545cb5)
看起来像是最后一个用户断开连接后会触发
看起来不影响后续链接新房间

**Comment by xfgryujk** [comment, marked as answer]
把翻译关掉

**Reply by SSSSSSSan** [reply]
收到！
